todo-facebookoauth/FacebookOAuth/www/scripts/facebookSignIn.js

```diff
@@ -0,0 +1,42 @@
+﻿var app = {
+
+    login: function () {
+        if (!window.cordova) {
+            var appId = prompt("Enter FB Application ID", "");
+            facebookConnectPlugin.browserInit(appId);
```

Comment:
I really don't love the name of this global "facebookConnectPlugin."  When I was working on Facebook (admittedly this was in ~2011) for my day job, the name of the global that Facebook added was "FB".